metrics/config_observability.go

```diff
@@ -127,3 +139,27 @@ func ConfigMapName() string {
 	}
 	return "config-observability"
 }
+
+// Use the same as `metrics.reporting-period-seconds` for the default
+// of `metrics.request-metrics-reporting-period-seconds`
+func getDefaultRequestMetricsReportingPeriod(data map[string]string) (*int, error) {
```

Comment:
Don't think it's necessary to return a `*int` - I think an `int` would suffice since we always return a default value